src/fprime_gds/common/loaders/fw_type_json_loader.py

```diff
@@ -0,0 +1,61 @@
+"""
+fw_type_json_loader.py:
+
+Loads flight dictionary (JSON) and returns name based Python dictionaries of Fw types
+
+@author jawest
+"""
+
+from fprime_gds.common.loaders.json_loader import JsonLoader
+from fprime_gds.common.data_types.exceptions import GdsDictionaryParsingException
+
+
+class FwTypeJsonLoader(JsonLoader):
+    """Class to load python based Fw type dictionaries"""
+
+    TYPE_DEFINITIONS_FIELD = "typeDefinitions"
+    
+    # List of Fw type names in dictionary
+    FW_TYPE_NAMES = [
+        "FwPacketDescriptorType",
+        "FwChanIdType",
+        "FwEventIdType",
+        "FwOpcodeType",
+        "FwTlmPacketizeIdType"
+    ]
+
+    def construct_dicts(self, _):
+        """
+        Constructs and returns python dictionaries keyed on id and name
+
+        Args:
+            _: Unused argument (inherited)
+        Returns:
+            A tuple with two Fw type dictionaries (python type dict):
+            (id_dict, name_dict). The keys should be the type id and
+            name fields respectively and the values should be type name
+            strings. Note: An empty id dictionary is returned since there 
+            are no id fields in the Fw type alias JSON dictionary entries.
+        """
+        id_dict = {}
+        name_dict = {}
+
+        if self.TYPE_DEFINITIONS_FIELD not in self.json_dict:
+            raise GdsDictionaryParsingException(
+                f"Ground Dictionary missing '{self.TYPE_DEFINITIONS_FIELD}' field: {str(self.json_file)}"
+            )
+
+        for type_def in self.json_dict[self.TYPE_DEFINITIONS_FIELD]:
+            try:
+                if type_def["kind"] == "alias" and type_def["qualifiedName"] in self.FW_TYPE_NAMES:
```

Comment:
Would it make sense to match on whether the name starts with `Fw`, rather than maintaining a list?

Reply:
And maybe it being unqualified (contains no `.`)  ?

Reply:
The GDS doesn't need them for now, but we could load them nonetheless. That way no need to maintain the list, and who knows maybe a user developing a GDS extension might need the information for some reason.

Reply:
Done!